examples/pipelines/kritis.yaml

```diff
@@ -47,9 +45,24 @@ spec:
         - name: integration-test          # 4. Run Integration Tests in test cluster
           taskRef:
               name: integration-test-in-docker
-          sourceBindings:
-              - inputName: workspace
+          inputSourceBindings:
+              - name: workspace
                 sourceKey: kritis
           params:
               - name: testArgs
                 value: "-e REMOTE_INTEGRATION=true"
+    sources:
```

Comment:
in the current design, this would make sense to put into the [PipelineParams](https://github.com/knative/build-pipeline#pipelineparams), i.e. replacing [these sources in the example](https://github.com/knative/build-pipeline/blob/master/examples/pipelineparams.yaml#L8)

Reply:
I wanted to make them outside of the `tasks` so it can be used across multiple tasks, but it would work either outside or in the PipelineParams

Reply:
I think we'd definitely want this in PipelineParams, unless we decide in the long run to do away with PipelineParams